forum/users/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
+from .serializers import CustomTokenObtainPairSerializer
 
 app_name = 'users'
 
 urlpatterns = [
-    
+    path('sign-in/', TokenObtainPairView.as_view(), name='sign_in'),
+    path('token/refresh/', TokenRefreshView.as_view(), name='roken_refresh'),
```

Comment:
1. Correct the name of the refresh path from 'roken_refresh' to 'token_refresh'.

2. It seems we already have an endpoint to refresh tokens provided by Djoser, like `/auth/jwt/refresh/` (refer to http://127.0.0.1:8000/swagger/). This might result in two endpoints with the same functionality. Please test both endpoints, choose the one that works best for you, and keep it.

Reply:
You're right, I fixe typo. Although, about endpoint - I think it's nice to keep them together, and maybe we don't really need all endpoints that djoser provides (all beside JWTs), at least rn

Reply:
Also in latest commits I moved those routes to separate list, that is accessed with prefix 'auth/'